app/controllers/concerns/hyrax/collections/accepts_batches.rb

```diff
@@ -36,7 +36,7 @@ def filter_docs_with_edit_access!
 
         def filter_docs_with_access!(access_type = :edit)
           no_permissions = []
-          if batch.empty?
+          if batch.empty? && flash[:notice].nil?
```

Comment:
Why the check for nil here? When is the notice ever not nil?

Reply:
This method gets called when drawing the dashboard for displaying all collections or my collections.  If you delete an Admin Set, flash[:notice] will contain: "Administrative set successfully deleted" when the code gets here, and you want this message to show-up, not "Select something first".  So this check makes sure the flash notice message does not get overwritten.

Reply:
This method (`filter_docs_with_access!`) is called in a before filter, so no flash could be set at this point, right?  If you revert this change is there a test that will fail that can illustrate why this conditional is necessary?

Reply:
Even though this is a before action, it is passed a notice in the redirect, and you want that notice to get out to the user:

`redirect_to hyrax.dashboard_collections_path, notice: t(:'hyrax.admin.admin_sets.delete.notification')`

I can't get a test to fail if I remove the checkout, but I do see this behavior I described above when I test it.

Reply:
I found a way not to have to do this check.  going to push a new pr for it now.